docs/Migrating-v4.md

```diff
@@ -339,24 +345,14 @@ EX: `{% include '@ecl/form-group/form-group.html.twig' with {
 - Tags now wrap on several lines by default. New twig parameter `nowrap` added to force display of the tag on one line.
 - Added possibilty to handle set of tags. Twig template and corresponding css have been added.
 
-## Custom theme
+## Customizing the display
 
 ECL4 introduces new ways to customize the look and feel of elements, by changing styles and component display
 
-### How to create a custom theme
-
-[To be done]
-
-### How to customize styles
-
-#### Using SASS variables
+### Using CSS variables
 
-[to be done]
-
-#### Using CSS variables
-
-CSS variables are provided for some styles values.
-You can find the full list of global CSS variables at the end of the file `src/themes/[your_theme]/_variables.scss`.
+The easiest way to change the look and feel of ECL is by using the provided CSS variables. They are provided for most common styles values.
```

Comment:
can we use custom properties instead of variables everywhere we relate to an --variable ?  Variables can be confusing, while custom properties are specifically identifying what we mean.